docs/section10/10-3.md

```diff
@@ -0,0 +1,40 @@
+# Airside systems – Rule 10-3
+
+**Rule ID:** 10-3
+ 
+**Rule Description:** For systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.  
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. The hvac system in B-RMR serves a computer room.
+
+**Function Call:** 
+
+1. get_zones_computer_rooms
+
+**Manual Check:** Check that the applicable baseline HVAC systems (hvac systems that serve comuter rooms) were modeled per the rules of G3.1-10 HVAC System baseline column which states that for systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.  
+ 
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of applicable zones in P_RMR: `applicable_zones_comp_room_p = get_zones_computer_rooms(P_RMR)`
+- For each zone in P_RMR: `zone_p in P_RMR...zones:`
+    - Check if the zone includes a computer room, if so then applicability flag is true: `if zone_p.id in applicable_zones_comp_room_p: rule_applicability_flag = TRUE`
```

Comment:
As commented in the function 'get_zones_computer_rooms', having a more intutive name than 'rule_applicability_flag' would be helpful. This could be something like 'zone list computer room'

Reply:
Addressed